autogen/agentchat/contrib/capabilities/teachability.py

```diff
@@ -246,10 +246,16 @@ class MemoStore:
     Vector embeddings are currently supplied by Chroma's default Sentence Transformers.
     """
 
-    def __init__(self, verbosity, reset, path_to_db_dir):
+    def __init__(
+        self,
+        verbosity: Optional[int] = 0,
+        reset: Optional[bool] = False,
+        path_to_db_dir: Optional[str] = "./tmp/teachable_agent_db",
```

Comment:
Not clear to me these are actually optional. Looks like the code expects them to be values

Reply:
yes, they are optional. if not provided by user, will be set to default.

Reply:
You both have good points. In the original code, these MemoStore parameters were not optional (even though the docstring said they were). But MemoStore was only instantiated by `Teachability.__init__`, which received the same parameters from its instantiator, where the parameters were truly optional. Nowadays people have cases where they instantiate MemoStore apart from Teachability, so I think it's an improvement to make these parameters optional at both levels.

Reply:
thanks for the review @rickyloynd-microsoft